pkg/api/types.go

```diff
@@ -887,6 +887,11 @@ type QuobyteVolumeSource struct {
 	// Default is no group
 	// +optional
 	Group string
+
+	// Name of Quobyte Tenant that the Volume is part of
+	// Default is DEFAULT
```

Comment:
why "DEFAULT"? link to quobyte doc would be helpful if this is a well-known name to quobyte